### ISSUE TITLE
[Enhancement][a11y] Доступность `File`

### ISSUE THREAD

**Issue by eugpoloz** [VKCOM/VKUI#2562]
- [ ] документация по доступности (https://github.com/VKCOM/VKUI/issues/3035)

--------

Использование визуального скрытого _<input type="file" />_ в https://github.com/VKCOM/VKUI/pull/2527 может приводить вот к таким интересным спецэффектам (на примере VoiceOver):

![image](https://user-images.githubusercontent.com/22026957/169814740-e46aa4f9-0f6a-4bbc-a8ae-4344229fccbc.png)

**Аудит невизуальной доступности** также обращает на это внимание:

> Отдельная надпись для скринридера "Выберите файл" не нужна, потому что браузер самостоятельно подписывает эту кнопку на языке системы.

Нужно изучить подходы к доступным `<input type="file" />` и сделать по красоте. :wink:

**Comment by SevereCloud** [comment]
### Дерево доступности

<details>
  <summary>Google Chrome</summary>
<img width="490" alt="image" src="https://user-images.githubusercontent.com/14944123/169848352-f6082e1d-a417-4ac7-b505-0415fc1dd8f7.png">
</details>

<details>
  <summary>Firefox</summary>
<img width="521" alt="image" src="https://user-images.githubusercontent.com/14944123/169856138-96c13ea3-6836-4d7b-9e68-27366303a4c1.png">
</details>

### Дерево доступности с выбранным файлом
<details>
  <summary>Google Chrome</summary>
<img width="481" alt="image" src="https://user-images.githubusercontent.com/14944123/169857146-61b5aa8d-b14f-4eeb-b2fb-8f1c9c00f566.png">

_ничего не изменилось_
</details>

<details>
  <summary>Firefox</summary>
<img width="465" alt="image" src="https://user-images.githubusercontent.com/14944123/169857383-7a70590e-761c-4c3e-a2f8-1d97b785953f.png">
</details>

## Имя файла

> Скрывать инпут вообще не вариант, потому что без инпута пользователь сейчас имя прикрепленного файла никак не узнает

- File это кнопка и в ней мы не показываем название файла
- Google Chrome игнорирует название файла
<details>
  <summary>Пруф</summary>
<img width="375" alt="image" src="https://user-images.githubusercontent.com/14944123/169859572-c9c9f911-c0f9-4cd8-816e-ca8f80740b57.png">
<img width="347" alt="image" src="https://user-images.githubusercontent.com/14944123/169859501-b8c02322-f50a-4716-9819-86db3912e287.png">
</details>
 
**предложение:** Просто добавить `aria-hidden`. Остальное оставить на разработчиков

**Comment by samohovets** [comment]
Здравствуйте. Пользователи прислали два бага:

https://vk.com/bug1027954
https://vk.com/bug1026403

Нативный попап перекрывает кнопку:
![image](https://github.com/VKCOM/VKUI/assets/24735429/b38fd68f-a519-456a-bf99-b6bc002d7a64)

Версия VKUI: 5.10.0

Рендерю вот так:

```js
<FormItem top="Детское новогоднее фото">
     <Spacing size={4} />
     <File
                required
                accept="image/*"
                name="photo"
                before={<Icon24Camera role="presentation" />}
                size="m"
      >
                Выбрать фотографию
      </File>
</FormItem>
```

При сабмите формы возникает вышеописанная проблема.

Видимо связано с этим issue :)

**Comment by BlackySoul** [comment]
> Здравствуйте. Пользователи прислали два бага:
> 
> https://vk.com/bug1027954 https://vk.com/bug1026403

> 
> При сабмите формы возникает вышеописанная проблема.
> 
> Видимо связано с этим issue :)

Привет! Завела на этот счет отдельную [issue](https://github.com/VKCOM/VKUI/issues/6283), потому что это касается в целом наших некоторых элементов форм.

**Comment by vkcom-publisher** [comment]
✅ <a href="https://github.com/VKCOM/VKUI/releases/tag/v6.0.2" target="_blank">v6.0.2</a> 🎉
